From 0516e95025bd0845abadb8cfcf3ab69e66928b84 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Wed, 18 Jun 2025 11:17:36 +0200
Subject: [PATCH 872/954] refactor(stm32mp23-fdts): remove pinctrl DT file
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

stm32mp23-pintctrl.dtsi file is unused and useless.

This reverts commit c1411b68da86c5e20b1861552670b01b29660269.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 2111721ca97f29345f68cc3e2791d5352d726c31 refactor(stm32mp23-fdts): remove pinctrl DT file]
Change-Id: Icba4bf80e3dd522dfc04c0c16a86833603724320
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/467954
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 fdts/stm32mp23-pinctrl.dtsi | 86 -------------------------------------
 1 file changed, 86 deletions(-)
 delete mode 100644 fdts/stm32mp23-pinctrl.dtsi

diff --git a/fdts/stm32mp23-pinctrl.dtsi b/fdts/stm32mp23-pinctrl.dtsi
deleted file mode 100644
index 4a2c8411be..0000000000
--- a/fdts/stm32mp23-pinctrl.dtsi
+++ /dev/null
@@ -1,86 +0,0 @@
-// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
-/*
- * Copyright (C) 2024, STMicroelectronics - All Rights Reserved
- * Author: Alexandre Torgue <alexandre.torgue@foss.st.com> for STMicroelectronics.
- */
-#include <dt-bindings/pinctrl/stm32-pinfunc.h>
-
-&pinctrl {
-	/omit-if-no-ref/
-	i2c7_pins_a: i2c7-0 {
-		pins1 {
-			pinmux = <STM32_PINMUX('D', 15, AF10)>, /* I2C7_SCL */
-				 <STM32_PINMUX('D', 14, AF10)>; /* I2C7_SDA */
-			bias-disable;
-			drive-open-drain;
-			slew-rate = <0>;
-		};
-	};
-
-	/omit-if-no-ref/
-	sdmmc1_b4_pins_a: sdmmc1-b4-0 {
-		pins1 {
-			pinmux = <STM32_PINMUX('E', 4, AF10)>, /* SDMMC1_D0 */
-				 <STM32_PINMUX('E', 5, AF10)>, /* SDMMC1_D1 */
-				 <STM32_PINMUX('E', 0, AF10)>, /* SDMMC1_D2 */
-				 <STM32_PINMUX('E', 1, AF10)>, /* SDMMC1_D3 */
-				 <STM32_PINMUX('E', 2, AF10)>; /* SDMMC1_CMD */
-			slew-rate = <2>;
-			drive-push-pull;
-			bias-disable;
-		};
-		pins2 {
-			pinmux = <STM32_PINMUX('E', 3, AF10)>; /* SDMMC1_CK */
-			slew-rate = <3>;
-			drive-push-pull;
-			bias-disable;
-		};
-	};
-
-	/omit-if-no-ref/
-	sdmmc2_b4_pins_a: sdmmc2-b4-0 {
-		pins1 {
-			pinmux = <STM32_PINMUX('E', 13, AF12)>, /* SDMMC2_D0 */
-				 <STM32_PINMUX('E', 11, AF12)>, /* SDMMC2_D1 */
-				 <STM32_PINMUX('E', 8, AF12)>, /* SDMMC2_D2 */
-				 <STM32_PINMUX('E', 12, AF12)>, /* SDMMC2_D3 */
-				 <STM32_PINMUX('E', 15, AF12)>; /* SDMMC2_CMD */
-			slew-rate = <2>;
-			drive-push-pull;
-			bias-pull-up;
-		};
-		pins2 {
-			pinmux = <STM32_PINMUX('E', 14, AF12)>; /* SDMMC2_CK */
-			slew-rate = <3>;
-			drive-push-pull;
-			bias-pull-up;
-		};
-	};
-
-	/omit-if-no-ref/
-	sdmmc2_d47_pins_a: sdmmc2-d47-0 {
-		pins {
-			pinmux = <STM32_PINMUX('E', 10, AF12)>, /* SDMMC2_D4 */
-				 <STM32_PINMUX('E', 9, AF12)>, /* SDMMC2_D5 */
-				 <STM32_PINMUX('E', 6, AF12)>, /* SDMMC2_D6 */
-				 <STM32_PINMUX('E', 7, AF12)>; /* SDMMC2_D7 */
-			slew-rate = <2>;
-			drive-push-pull;
-			bias-pull-up;
-		};
-	};
-
-	/omit-if-no-ref/
-	usart2_pins_a: usart2-0 {
-		pins1 {
-			pinmux = <STM32_PINMUX('A', 4, AF6)>; /* USART2_TX */
-			bias-disable;
-			drive-push-pull;
-			slew-rate = <0>;
-		};
-		pins2 {
-			pinmux = <STM32_PINMUX('A', 8, AF8)>; /* USART2_RX */
-			bias-disable;
-		};
-	};
-};
-- 
2.53.0

